docs/howto_config.md

```diff
@@ -24,6 +24,7 @@ node:
 apiServer:
   subjectAltNames:
     - ""
+  advertiseAddress: ""
```

Comment:
This is exposed so as to enable CNCF certification tests where we need more than one node. Users shouldnt use it, as it defaults to the api server service IP internally.

Reply:
If we put this in the documentation, someone is going to use it. Do we need to include it in this document?

Reply:
That's a good question. @jerpeter1 has been looking at the certification rules, maybe he came across something that would help answer it?